sdks/java/core/src/main/java/org/apache/beam/sdk/coders/Coder.java

```diff
@@ -198,6 +198,9 @@ public static void verifyDeterministic(Coder<?> target, String message, Iterable
     }
   }
 
+  public static <T> long getEncodedElementByteSizeUsingCoder(Coder<T> target, T value) throws Exception {
```

Comment:
We probably don't need the UsingCoder suffix, the signature is different so the overload will be unambiguous.

Reply:
Looks like Java doesn't allow this.